src/generator/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+
+const jsGenerator = require('./techs/javascript');
+const generalGenerator = require('./techs/general');
+
+/*
+ * Returns js-code that requires BEM entities
+ */
+module.exports = bemDeps => {
+    const techToFiles = {};
+
+    bemDeps.forEach(({ path, tech }) => {
+        techToFiles[tech] = (techToFiles[tech] || []).concat(path);
+    });
+
+    return Object
+        .keys(techToFiles)
+        .sort(tech => tech === 'js') // js should be the last
+        .map(tech => {
+            const generator = tech === 'js' ? jsGenerator : generalGenerator;
+            const existingFiles = techToFiles[tech].filter(fs.existsSync);
```

Comment:
ммм, а нельзя это делать через webpack-овские ручки?
странно работать с фс напрямую, когда webpack должен всё контролировать

Reply:
Hmm. In my solution "No".

I do resolve bem-require `b:b e:e` by module implemented with source `(require('b.js'),require('b_e.js')).applyDecls()`. Lack of files causes module require error. To handle this we need to override requires of actual files. I want to avoid this and reduce amount of redefinitions of standard webpack mechanisms.

I can add a comment in a way "TODO: should do it more webpackable"